code/modules/mob/living/living.dm

```diff
@@ -81,11 +81,7 @@
 		to_chat(src, "<font color='blue'>You are not injured enough to succumb to death!</font>")
 
 /mob/living/proc/updatehealth()
-	if(status_flags & GODMODE)
```

Comment:
GODMODE is unmaintained and therefore deprecated, so when I happen upon references I axe them.